sdk/cosmos/azure-cosmos/docs/TimeoutAndRetriesConfig.md

```diff
@@ -0,0 +1,31 @@
+## Cosmos DB Python SDK – Timeout configurations and Retry configurations
+
+### Timeout config
+
+The timeout options for the client can be changed from the default configurations with the options below. These are:
+- `Client Timeout`: can be changed by passing the `timeout` option. This option has no default value, and is only present
+if it is passed in by the user. If not present, the other connectivity timeouts will be used.
+- `Request Timeout`: can be changed by passing the `request_timeout` option. Default value is 60s.
+- `Connection Timeout`: can be changed through `connection_timeout` option. Default value is 60s.
+- `Read Timeout`: can be changed through `read_timeout` option. Default value is 300s.
+
+
+### Resource Throttle Retry Policy config
+
+The options for the resource throttle retry policy used for 429 error codes can be changed from the default client configurations with the options below. These are:
+- `Retry Total`: can be changed by passing the `retry_total` option. Default value is 9.
+- `Retry Fixed Interval`: can be changed by passing the `retry_fixed_interval` option. This option has no default value,
+and is only present if it is passed in by the user.
+- `Retry Backoff Max`: can be changed by passing the `retry_backoff_max` option. Default value is 30s.
+
+
+### Connection Retry config
+
+The retry options below can be changed from the default client configurations. These are:
+- `Retry Connect`: can be changed by passing the `retry_connect` option. None.
+- `Retry Read`: can be changed by passing the `retry_read` option. None.
+- `Retry Status`: can be changed by passing the `retry_status` option. None.
```

Comment:
I'm not sure what the trailing `None`. means here. Is the implication that there is "no default"?
Because that's not really true - we retry by default in these scenarios under particular conditions. We don't want to imply that unless these parameters are passed in, no retries will be attempted.